viewer/src/views/threejs/cad/materials.ts

```diff
@@ -22,9 +23,17 @@ export interface Materials {
   instancedMesh: THREE.ShaderMaterial;
   triangleMesh: THREE.ShaderMaterial;
   simple: THREE.ShaderMaterial;
+  // Data textures
+  colorDataTexture: THREE.DataTexture;
```

Comment:
Is `overrideColorPerTreeIndex` or something similar more describing of the layout and purpose?